specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Pipeline.json

```diff
@@ -1957,6 +2745,10 @@
         "collectionReference": {
           "type": "object",
           "description": "The JSON Path of the Nested Array that is going to do cross-apply. Type: object (or Expression with resultType object)."
+        },
+        "mappings": {
+          "type": "object",
+          "description": "Column mappings with logical types"
```

Comment:
The description is longer in private repo, please copy from there. Also, there are two more description updates for schemaMappings and columnMapping property, where we advise the customers not to use them and use this instead. Please sync with David.